README.md

```diff
@@ -0,0 +1,77 @@
+# Maven Plugin Integration Testing
+
+Java library for writing integration tests for maven plugins.
+
+## Features
+
+* Run tests with isolated local repository (the plugin won't be installed in your default local maven repository)
+* Extract code coverage
+* Debugging
+
+## Usage
+
+```java
+import java.io.IOException;
```

Comment:
If you want to provide imports here, maybe you can also provide other imports too? I think this is not the only import you need to use the code below

Reply:
I removed it.